src/ontology/mondo-edit.obo

```diff
@@ -371316,9 +371316,11 @@ xref: Orphanet:94063 {source="MONDO:equivalentTo"}
 xref: SCTID:719046005 {source="MONDO:equivalentTo"}
 xref: UMLS:C4305140 {source="MONDO:equivalentTo"}
 xref: UMLS:CN206727 {source="MONDO:equivalentTo"}
-is_a: MONDO:0000508 {source="Orphanet:94063"} ! syndromic intellectual disability
+is_a: MONDO:0000761 {source="Orphanet:94063"} ! syndrome caused by partial chromosomal deletion
```

Comment:
We only want to attribute superclasses to a source if the source ontology asserts this superclass. In this case, Orphanet does not assert this superclass, so this assertion should be deleted.